17.md

```diff
@@ -0,0 +1,111 @@
+NIP-17
+======
+
+Git Commits and Discovery Over Nostr
+------------------------------------
+
+`draft` `optional` `author:melvincarvalho`
+
+- Reserves:  `kind 17`
+
+Do not merge until implemented, work in progress
+------------------------------------------------
+
+Status: This NIP is an early draft and work in progress.  Implementations are being developed which will incorporate what is learnt.
+
+
+Introduction
+------------
+
+Git Commits over nostr enable **uses cases** similar to:
+
+- [github actions](https://github.com/features/actions)
+- [continuous integration](https://en.wikipedia.org/wiki/Continuous_integration)
+- [continuous deplyment](https://en.wikipedia.org/wiki/Continuous_deployment)
+- [travis](https://travis-ci.org/)
+
+but without central points of failure.  The aim is to create continuous distributed software development processes, without having to rely on complex or centralized services.
+
+**You need only know the nostr ID or bech32 form of a repo you will be able to discover all versions, its origins and its history.**
+
+The git object data is not designed to be stored in nostr, but rather, the signaling, update, version and commit data.  Git data will live in existing git repositories, github, gitlab, gitea, gogs, ssh, file system etc.
+
+
+New Kind
+========
+
+A special event with kind `17`, can be used to indicate git commit messages. This number was selected to be hopefully easy to remember : `G-1-7`.  
+
+
+New Tag
+=======
+
+A new tag "c" will be used to store commit hashes from git a new version of the repo 
+
+
+Bech32
+======
+
+The prefix `nrepo` can be used in front ends that support it, to allow easier copy and paste
```

Comment:
An example would be great here. What comes after nrepo?

Reply:
Is the same as npub really